configuration/controller/rsyslog.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+######################################################################
+# TrinityX
+# Copyright (c) 2016  ClusterVision B.V.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License (included with the sources) for more
+# details.
+######################################################################
+
+
+set -e
```

Comment:
Please do not use ``set -e``. This is a debug feature, which is set for you when you use the ``--hardstop`` flag of the configuration tool. ``set -e`` prevents proper error management, and will fail on any command that returns a non-zero error code, including when that error code is expected. Those issues are well documented.
https://github.com/clustervision/trinityX/blob/development/doc/config_post_scripts.rst